### DISCUSSION TITLE
Example GenAI bot with guardrails

### DISCUSSION THREAD

**Discussion by wangxj03** [langfuse/langfuse#3003] [Support]
Langfuse is great to check latency and intermediate results!

Blog post: https://wangxj03.github.io/posts/2024-08-18-guardrails/
Fully reproducible code: https://github.com/wangxj03/ai-design-patterns/tree/main/guardrails

Feedback and comments welcome!

**Comment by marcklingen** [comment]
thanks for sharing, this was a great read

**Reply by SalesAdminTechDev** [reply]
Great post.
